procedures/unit-testing-basics.ipf

```diff
@@ -667,6 +667,29 @@ static Function UTF_ToSystemErrorStream(message)
 	systemErr += message + "\r"
 End
 
+/// Reports an error. This is used for internal UTF Hooks.
```

Comment:
The description should state a bit clearer why this special function must be used from hooks to report errors.
Also does it work from all UTF hooks?